RELEASE-NOTES.txt

```diff
@@ -8,6 +8,7 @@
 * [**] [internal] Disable Story posts when Jetpack features are removed [#19823]
 * [*] [internal] Editor: Only register core blocks when `onlyCoreBlocks` capability is enabled [https://github.com/wordpress-mobile/gutenberg-mobile/pull/5293]
 * [**] [internal] Disable StockPhoto and Tenor media sources when Jetpack features are removed [#19826]
+* [*] [Jetpack-only] Fixed a bug where analytics calls weren't synced to the user account. [#19926]
```

Comment:
Just for info, were the author informed that this item added during beta-testing phase will not make it to the editorialized release notes in time? (even if it's still nice to have added them in this `RELEASE-NOTES.txt` anyway just to keep track of things)

Reply:
Yes. See https://github.com/wordpress-mobile/WordPress-iOS/pull/19926#issuecomment-1386048901

Reply:
Perfect! 👌